pipeline/frontend/yaml/parse_test.go

```diff
@@ -275,6 +275,7 @@ func TestReSerialize(t *testing.T) {
             - test.zig
         - path:
             exclude: a
+            on_empty: true
```

Comment:
yes this refactor is also a bugfix :)

Reply:
well i think we should let serialization clean up config so did 1a7d7c4faf8c9f934b7f5129d872f73db21d0fb7 to restore old behaviour